Automattic/newspack-plugin
# fix(media-credit): refactor clauses for credit search

### All Submissions:






### Changes proposed in this Pull Request:

Refactors the media credit search feature to use custom query clauses in order to avoid conflicting with other plugins that may add other joins to the query.

### How to test the changes in this Pull Request:

1. Add credit metadata to an attachment
2. Make sure you have Jetpack's VideoPress active
3. Visit the Media Library and search using the credit value
4. Confirm the search returns the attachment
5. Written tests should continue to pass

### Other information:
